content/package.json

```diff
@@ -17,11 +17,11 @@
   },
   "dependencies": {
     "@dcl/catalyst-api-specs": "1.2.2",
-    "@dcl/catalyst-contracts": "1.0.2",
-    "@dcl/catalyst-node-commons": "1.0.4",
-    "@dcl/content-validator": "1.5.0",
+    "@dcl/catalyst-contracts": "^2.0.0",
+    "@dcl/content-validator": "^1.5.1",
+    "@dcl/crypto": "^3.0.0-20220530200011.commit-cef28ff",
```

Comment:
Use release version

Reply:
Done.

What is the deal with main branch versions? I'm observing this is common part of the reviews but I don't get why it may be a problem

Reply:
🤔  Well, not sure what's the npm retention policy for this kind of builds `^3.0.0-20220530200011.commit-cef28ff"`.